ntbs-service/Pages/Search/Index.cshtml.cs

```diff
@@ -53,18 +72,17 @@ public async Task<IActionResult> OnGetAsync(int? pageIndex)
 
         public IQueryable<Notification> OrderQueryable(IQueryable<Notification> query) {
             return query.OrderByDescending(n => n.CreationDate)
-                .OrderByDescending(n => n.SubmissionDate)
-                .OrderBy(n => n.NotificationStatus);
+                .OrderByDescending(n => n.SubmissionDate);

```

Comment:
Just a general thought, not something to take action on yet. We should make sure that CreationDate is just the Date and not DateTime. Otherwise, the second order function will be unreachable